### DISCUSSION TITLE
Redirect Bug in Vercel Dashboard

### DISCUSSION THREAD

**Discussion by DevSrijit** [vercel/community#5784] [Bug Report]
### Page to Investigate

https://vercel.com/[username]/~/stores

### Hi,
I found a replicable bug in Vercel Storage Dashboard. Here are the steps to replicate the bug:

- Create an Edge Config Storage
- Go to settings
- Delete the Edge Config db
- Now, you are redirected to `https://store/` which is an invalid route.

TLDR; On deleting an edge config db, we are being routed to `https://store/`, which is a bug.

Here's a screen recording:

https://github.com/vercel/community/assets/89468732/8c4b414f-7c1c-4b2e-b54c-d9044796ff18

**Comment by amyegan** [comment]
Thanks for sharing these details, @DevSrijit. I'll let the team know about it so we can get it fixed.

**Reply by DevSrijit** [reply]
Great, thanks for looking into it.

**Reply by amyegan** [reply]
This is fixed now. Thanks again for reporting it!
